web-ts/src/RootSwitcher.tsx

```diff
@@ -62,22 +63,26 @@ const RootSwitcher = ({ setContentLanguage }: PropsType): ReactElement => {
       </Layout>
     )
   }
+  const relevantCities = fixedCity ? cities.filter(city => city.code === fixedCity) : cities
 
   return (
     <Switch>
-      <Route exact path={RoutePatterns[LANDING_ROUTE]} render={props => <LandingPage cities={cities} {...props} />} />
+      <Redirect exact from='/' to={fixedCity ? cityContentPath : landingPath} />
+      <Redirect exact from={`/:cityCode`} to={cityContentPath} />
+      <Redirect exact from={`/${LANDING_ROUTE}`} to={landingPath} />
+      {fixedCity && <Redirect exact from={RoutePatterns[LANDING_ROUTE]} to={landingPath} />}
```

Comment:
What is this used/needed for? It seems a little bit weird :see_no_evil:

Reply:
yeah. I seem to have mixed up something here ^^